msal/src/MSAL.Common.props

```diff
@@ -98,16 +98,19 @@
     <None Include="$(PathToCoreSources)\*.md" LinkBase="Core" />
     <Compile Remove="$(PathToCoreSources)\Platforms\**\*.cs" />
     <Compile Remove="$(PathToCoreSources)\Features\**\*.*" />
+    
+    <!-- Any feature that is included in NetStandard MUST be included in ALL frameworks. 
+    Non-implementing frameworks should throw a PlatformNotImplementedEx and hide the public surface at build time-->
+    <Compile Include="$(PathToMsalSources)\Features\PublicClientWithTokenCache\**\*.cs" LinkBase="Features\PublicClientWithTokenCache" />
+    
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.3'">
     <Compile Include="$(PathToMsalSources)\Platforms\netstandard1.3\**\*.cs" LinkBase="Platforms\netstandard1.3" />
     <Compile Include="$(PathToCoreSources)\Platforms\netstandard1.3\**\*.cs" LinkBase="Core\Platforms\netstandard1.3" />
     <Compile Include="$(PathToCoreSources)\Features\Cache\**\*.cs" LinkBase="Core\Features\Cache" />
-    <Compile Include="$(PathToMsalSources)\Features\PublicClientWithTokenCache\**\*.cs" LinkBase="Features\PublicClientWithTokenCache" />
```

Comment:
I moved this 2 files in a single directory because they represent a single feature. We should not expose a PCA ctor 
 with TokenCache without also exposing Serialize, Deserialize, AfterAccess etc.